### ISSUE TITLE
Inspector error in cider-inspect

### ISSUE THREAD

**Issue by reedho** [clojure-emacs/cider#1005]
Doing cider-inspect show error like below in the inspect buffer:

    Inspector error for: "(\"Type\" \": \" (:value ...

Quick investigation on the *nrepl-message* show that it seems the string value returned by the middleware is double printed, like below: 

```lisp
(<-
  id  "120"
  session  "df3ba8e0-c181-4a9b-a304-d19068da9420"
  status  ("done")
  value  "\"(\\\"Type\\\" \\\": \\\" (:value \\\"java.lang.Class\\\" 0) (:newline) (:newline) \\\"--- \\\" ... "
)
```

I thought, it also cause same error with `cider-browse-ns-all`.

**Comment by cichli** [comment]
Hmm, what's the value you're trying to inspect? This may be resolved by clojure-emacs/cider-nrepl#164, once #1009 is merged.

**Comment by reedho** [comment]
Hi @cichli, 

Error above was when using cider-nrepl-0.9.0-20150228.222204-12.jar, any values (except LazySeq) give "Inspector error for: ..."

Now with both latest cider (package: 20150308.900) and cider-nrepl (cider-nrepl-0.9.0-20150307.214951-14.jar), it works perfectly as you said.

Thank you very much.
